snakesist/exist_client.py

```diff
@@ -382,17 +382,24 @@ def query(self, query_expression: str) -> etree._Element:
         :return: The query result as a ``delb.Document`` object.
         """
 
-        params = {
-            "_howmany": "0",
-            "_indent": "no",
-            "_wrap": "yes",
-            "_query": query_expression,
-        }
-
-        response = requests.get(
+        request_body = (
```

Comment:
imo this assignment isn't necessary and the payload can be assembled in the `post`'s arguemnts expression.

a micro optimization *could* be to define this as a string 'constant' and use `str.format` for extrapolation.

Reply:
I like the micro optimization, will try it out.